src/indicators/flooding/glofas/utils.py

```diff
@@ -180,10 +201,25 @@ def get_crps(
     return df_crps
 
 
-def get_groups_above_threshold(observations, threshold, min_duration=1):
-    groups = np.where(np.diff(observations > threshold, prepend=False, append=False))[
-        0
-    ].reshape(-1, 2)
+def get_groups_above_threshold(
+    observations: ArrayLike,
```

Comment:
How would ```ArrayLike``` be different than a standard list/array?

Reply:
Oops I had not understood ArrayLike, I am putting it back to Array (ArrayLike is anything that can be coerced to an array)